src/client/app/effects/snippet.ts

```diff
@@ -61,6 +61,7 @@ export class SnippetEffects {
             const scrubbedSnippet = getScrubbedSnippet(rawSnippet, publicOrInternal);
             delete scrubbedSnippet.modified_at;
 
+            storage.snippets.load();
```

Comment:
Why do we need to load the entirety of the snippets?  What purpose does this code serve?

Reply:
// BUG number XYZ:  The load is just a workaround, look for a more performant solution later

Reply:
Can you also document the workaround here to keep track of things. Also please reference this PR in the bug.

Reply:
Added comment.